fuzz/README.md

```diff
@@ -0,0 +1,51 @@
+## build
```

Comment:
Please reorganize it according to the following format

# Building

## Build Requirements

Note that `lib-fuzzer` is part of `clang`, lib-fuzzer is xxxxx(please introduce lib-fuzzer simply)

Follow these steps to install clang:

- Debian / Ubuntu:
```sh
apt install -y xxxx
```

- CentOS / Fedora / RHEL:
```
yum install -y xxx
```

## Build and Install

Add some description in here。

```sh
cmake -DRATS_TLS_BUILD_MODE="host" -DBUILD_SAMPLES=on -DBUILD_FUZZ=on -H. -Bbuild
make -C build install
```
# FUZZ

## rats_tls_init API
## rats_tls_negotiate API
## rats_tls_transmit/recv/clean_up APIs